repos/spack_repo/builtin/packages/py_ndindex/package.py

```diff
@@ -18,3 +20,8 @@ class PyNdindex(PythonPackage):
     version("1.7", sha256="bf9bd0b76eeada1c8275e04091f8291869ed2b373b7af48e56faf7579fd2efd2")
 
     depends_on("py-setuptools", type="build")
```

Comment:
According to the file I downloaded, there should be a dependency on `python@3.7:` (since we still list `3.6`) and `c` (per `setup.py`'s `check_cython()`).